component/xload/stats_manager.go

```diff
@@ -210,11 +221,14 @@ func (sm *StatsManager) calculateBandwidth() {
 	filesPending := sm.totalFiles - filesProcessed
 	percentCompleted := (float64(filesProcessed) / float64(sm.totalFiles)) * 100
 	bandwidthMbps := float64(bytesTransferred*8) / (timeLapsed * float64(MB))
+	diskSpeedMbps := float64(sm.bytesWritten*8) / (timeLapsed * float64(MB))
```

Comment:
How much difference are you seeing between `bandwidthMbps` and `diskSpeedMbps`

Reply:
as of now I see them almost same, disk being little low compared to download. But these new stats cleared that there are no more blocks left in the blockpool and many data manager instances are waiting. This means most of the blocks are in the responce queue only, and that will happen only if disk writing is slower then download speed.